src/documents/IoCAPI.md

```diff
@@ -86,11 +87,11 @@ of view of component `that`.
 
 ### fluid.parseContextReference(reference) {
 
-Parse an [IoC reference](IoCReferences.md) of the form `"{myContext}.myPath1.myPath2"` into an object form. In this case, for example, the output would be 
+Parse an [IoC reference](IoCReferences.md) of the form `"{myContext}.myPath1.myPath2"` into an object form. In this case, for example, the output would be
 
 ```json
 {
-    context: "myContext", 
+    context: "myContext",
```

Comment:
should we fix these malformed snippets in the process, or do that later?

Reply:
Unless we can add in a JSON5 mode to the plugin :)
A semi-serious suggestion, I'd be willing to help out with this if it looked feasible.

Reply:
I mentioned to Antranig that there is a confusingly-named [`grunt-mdlint` plugin](https://github.com/ChrisWren/grunt-mdlint) that claims to lint javascript code within markdown.  This would help here if we used the `javascript` type for these blocks, but would also help with other blocks that are in fact javascript.

Unfortunately, after trying the plugin this morning, it seems like it wasn't updated for the jump from npm@2 to npm@3.  It tries to make a hard-coded call to the underlying mdlint binary that does not work if mdlint is already a dependency and in your project's node_modules instead of the plugin's:

```
Error: Cannot find module '/Users/duhrer/Source/rtf/infusion-docs/node_modules/grunt-mdlint/node_modules/mdlint/bin/mdlint'
```

I will see if we have the option to plug something into mdlint via the grunt-markdownlint plugin itself.

Reply:
There is [a plugin for ESLint](https://github.com/eslint/eslint-plugin-markdown), which seems interesting.  I'll give that a try.  I have not found anything comparable for JSON or JSON5 yet.

Reply:
OK, I redefined the `eslint` task as a multitask.  `eslint:js` is the previous default and runs as part of the `lint` rollup command.  You can use `grunt eslint:md` to see the errors reported when attempting to validate the blocks that are flagged as `javascript`.  I kept looking, but did not find any comparable plugin to validate `json` and `json5` blocks within a markdown file.